tests/integration/ethernet_mtu_test.py

```diff
@@ -27,10 +27,21 @@
 from .testlib.statelib import INTERFACES
 
 
-def test_increase_iface_mtu():
+@pytest.fixture(scope="function", autouse=True)
+def init_eth_iface():
     desired_state = statelib.show_only(('eth1',))
     eth1_desired_state = desired_state[INTERFACES][0]
     eth1_desired_state['state'] = 'up'
+    eth1_desired_state['ipv6'] = {'enabled': True}
+    # When a link came up, ipv6 link local address will be automatically
```

Comment:
It seems to me like you use this to "fix" a valid scenario.
You also changed the tests: They started from a state=down, and now they start from state=up.
When the iface is down, ipv6 is enabled=False and when the iface is taken up, the user cannot (currently) leave the ipv6 disabled.
The minimalistic change needed is in the tests: to set ipv6 enabled (like we do wit the state=up).
A fixture can be used to provide the base dict (with state=up and ipv6.enabled=True), but not perform an action on the system. If you do that, I would do it in a separate PR or commit.